sdf/1.7/light_state.sdf

```diff
@@ -1,5 +1,5 @@
 <!-- State information for a light -->
-<element name="light" required="*">
+<element name="light_state" required="*">
```

Comment:
I'm not sure if we should be changing these names because they are actually used by Gazebo-classic. If I remember correctly, keeping the original names causes an issue with the generated xsd. I was hoping we can fix that issue without renaming these elements.

Reply:
Renamed them back in aef3ab859c20cdc955c10fe48a11343966f28d27